Lib/test/test_interpreters/test_stress.py

```diff
@@ -32,6 +32,23 @@ def task():
         with threading_helper.start_threads(threads):
             pass
 
+    @support.requires_resource('cpu')
```

Comment:
I'm not sure if `requires_resource('cpu')` can pass without working threading, but adding that seemed to help fix an issue I was having with threading in CI the other day (see GH-126849)
```suggestion
    @support.requires_resource('cpu')
    @threading_helper.requires_working_threading()
```

Reply:
We get the same effect through other means. like, `import_helper.import_module()`, but there's no harm in being explicit about it.

Reply:
done

Reply:
Good to know :)